src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -122,7 +126,7 @@ protected override IWebUIFactory CreateWebUiFactory()
             return new AndroidWebUIFactory();
         }
 
-        protected override ICryptographyManager InternalGetCryptographyManager() => new AndroidCryptographyManager();
+        protected override ICryptographyManager InternalGetCryptographyManager() => new CommonCryptographyManager();
```

Comment:
Is this new to Android SDK 12, or is it available on earlier platforms?

Reply:
This object is internal to MSAL, it's not part of the SDK.